Authentication/Validators/TextInputValidatorType.swift

```diff
@@ -15,6 +15,10 @@ public enum ValidationResult {
         return .Invalid(errors: [errorMessage])
     }
     
+    public static func valid() -> ValidationResult {
```

Comment:
Are there any rules defined for such things in either the github styel guid or swift's?